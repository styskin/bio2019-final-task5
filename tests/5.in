9 10
1 31245
1 146317
1 229259
1 777573
2 256788
3 91162
3 107474
3 116740
3 442418
3 776647
50
1/1
0/1
1/0
0/0
1/1
1/1
0/0
0/0
1/1
1/1

1/0
1/1
1/1
0/0
0/1
1/0
0/0
0/0
0/0
1/0

0/1
1/1
0/1
1/1
1/1
1/1
0/0
0/0
0/1
1/0

0/1
0/0
0/1
0/0
0/1
1/0
0/1
0/0
0/1
0/0

1/1
0/0
0/1
1/1
1/1
0/1
1/0
1/1
0/0
0/0

1/1
1/1
1/1
1/1
0/1
1/0
1/0
1/0
0/1
1/1

0/0
1/1
0/1
1/0
1/0
1/1
1/1
0/1
1/1
1/1

1/0
0/1
1/1
0/1
1/0
0/0
0/1
1/0
1/1
1/1

0/0
0/0
1/0
0/1
0/1
0/0
1/1
0/1
0/1
1/0

0/1
1/0
1/0
0/0
1/1
0/0
0/1
1/0
1/1
1/1

0/1
1/0
1/1
1/0
0/0
0/0
1/1
1/0
1/0
0/0

0/0
0/0
1/0
1/0
1/1
0/0
1/0
0/1
0/0
1/0

0/1
0/1
0/0
1/1
1/0
1/0
0/1
0/0
1/1
0/1

1/1
0/1
0/0
1/0
1/1
1/1
0/1
0/1
1/0
1/1

0/0
0/0
0/0
0/0
1/0
0/1
1/1
1/1
0/1
1/1

1/1
0/0
1/1
1/0
0/0
1/1
0/0
0/0
0/0
1/0

0/1
0/1
0/0
1/0
1/1
1/1
0/0
1/1
1/0
1/1

1/0
1/1
1/1
0/0
1/0
0/0
1/1
0/0
1/1
0/1

1/1
1/1
1/1
0/1
1/0
1/1
1/0
1/0
1/0
0/0

1/0
1/1
1/0
0/0
1/0
0/0
0/1
0/0
0/1
0/1

1/0
0/1
1/0
0/1
1/1
1/0
0/0
0/0
1/1
0/1

0/1
0/0
0/0
1/1
1/0
1/0
1/0
0/0
0/1
0/1

1/0
1/0
1/1
0/1
0/1
1/1
1/0
1/1
0/0
1/0

1/0
0/0
0/1
1/1
0/1
0/0
1/0
0/0
1/0
1/1

1/1
0/0
1/0
1/1
1/1
1/0
1/1
1/0
0/1
0/0

1/0
1/0
1/0
0/0
1/1
0/1
0/0
1/0
0/1
0/1

1/0
1/0
1/1
0/0
1/1
1/0
0/1
1/0
0/1
0/1

1/1
0/1
0/0
0/0
1/1
0/1
1/0
1/1
1/1
0/1

0/1
0/0
0/0
0/1
1/1
0/1
0/0
1/1
1/0
1/1

0/0
0/0
1/1
0/1
0/0
0/0
0/1
0/1
0/0
1/1

1/1
0/1
1/1
0/1
1/1
1/0
1/1
0/1
0/0
0/1

0/1
0/1
1/0
1/0
1/1
1/0
0/0
1/0
1/0
1/1

0/0
1/1
1/0
0/0
1/0
0/1
1/0
1/0
1/0
1/1

1/0
0/1
1/0
1/1
1/1
0/0
0/0
1/0
0/0
0/0

1/1
0/0
0/0
1/0
0/0
0/1
0/1
1/0
0/0
0/0

0/1
0/0
1/1
1/1
0/1
1/0
1/1
0/1
0/1
0/1

1/0
1/1
1/1
1/0
1/1
1/0
0/1
0/1
1/1
1/1

0/1
1/0
1/0
1/1
0/1
0/1
0/1
1/0
1/1
1/0

1/0
1/1
0/1
0/1
1/0
0/0
0/0
1/1
0/0
1/0

0/0
1/0
0/1
1/1
0/1
1/1
1/1
1/1
1/0
0/0

1/0
1/1
1/1
0/1
0/0
0/0
0/1
1/1
1/1
1/1

0/1
1/1
0/0
0/1
0/1
1/0
1/0
1/0
1/1
1/0

0/0
0/1
1/1
1/1
0/1
1/1
1/1
1/1
1/0
0/0

0/1
1/0
1/0
0/1
0/1
0/1
0/1
0/0
0/1
1/1

0/0
0/1
1/0
0/1
0/0
0/0
0/1
0/0
1/1
0/1

0/1
0/1
0/0
1/0
1/1
0/1
0/0
0/0
0/1
1/1

0/1
0/1
1/1
1/0
0/1
1/0
0/0
0/1
0/0
0/1

1/0
0/0
1/0
1/0
1/1
1/1
1/1
0/0
1/0
0/0

0/1
1/0
1/1
0/1
0/0
1/1
1/0
1/0
1/1
1/0

1/0
0/1
1/0
1/1
1/0
0/0
0/0
0/1
0/1
0/0

0/0
0/1
1/1
0/0
1/0
0/0
0/1
1/1
0/1
1/0

1/0
1/1
1/0
0/1
0/1
0/0
0/1
0/1
0/1
1/0

0/0
1/0
1/0
0/0
0/1
1/1
0/0
1/0
0/1
1/0

1/0
0/0
0/0
0/1
0/0
0/1
0/1
0/0
1/1
0/1

1/0
0/0
1/0
0/0
0/0
1/1
1/0
0/0
0/0
1/1

1/0
1/1
0/1
0/0
1/0
0/0
0/1
0/1
1/1
0/1

1/1
1/1
1/1
1/1
0/0
0/1
1/0
1/0
1/0
1/1

1/1
1/0
0/0
1/1
0/0
0/1
1/0
1/1
0/1
1/0

1/0
0/1
1/1
1/0
0/0
0/1
0/1
1/1
1/0
1/0

1/1
0/1
1/1
1/1
1/0
0/0
1/0
0/1
0/1
0/1

0/0
1/0
0/0
1/1
1/1
1/0
1/0
0/1
0/1
1/1

1/0
0/1
1/1
1/1
0/1
1/1
0/1
0/1
1/0
1/1

0/0
0/0
0/1
0/1
0/1
0/0
0/1
1/0
1/1
0/1

0/0
0/1
0/1
1/1
0/0
1/0
1/0
1/1
0/0
0/0

1/0
1/0
1/1
0/0
1/0
1/0
1/0
1/0
0/1
1/1

0/1
0/1
1/0
0/1
1/1
0/0
1/1
1/0
1/1
1/1

0/0
0/0
1/1
1/1
0/1
1/1
1/1
1/1
0/1
1/0

1/1
1/1
1/0
1/1
1/0
0/0
1/1
1/1
1/0
0/1

1/1
0/1
0/0
1/1
0/1
0/1
1/1
0/0
0/1
1/1

0/0
1/1
1/0
1/1
1/1
0/1
1/0
1/1
1/0
0/0

0/1
0/1
0/0
1/0
0/1
0/0
1/1
1/1
1/1
0/0

0/0
0/1
0/1
0/0
0/1
1/0
1/1
1/0
0/0
0/0

1/0
0/0
1/1
1/1
0/1
1/0
0/0
1/1
1/0
1/1

0/1
0/1
0/1
0/0
0/1
1/0
0/0
0/0
1/0
0/1

1/0
0/0
1/0
1/1
1/1
0/0
1/0
0/1
1/1
0/0

1/0
1/1
1/0
1/0
0/1
1/0
0/1
1/0
0/1
1/1

1/1
0/1
0/0
0/0
1/1
1/0
1/1
0/0
1/0
0/1

1/1
0/1
0/0
1/1
1/1
0/1
1/0
0/1
0/0
1/1

0/0
0/0
1/1
0/0
1/0
0/0
0/1
0/0
1/0
0/0

1/0
1/0
1/1
1/1
0/1
1/0
0/0
1/1
1/1
1/0

0/0
0/1
1/0
0/1
0/1
0/1
0/0
1/1
1/1
1/0

1/0
0/1
0/1
0/0
1/0
0/0
0/1
0/1
1/1
0/1

1/0
0/0
1/0
0/0
1/1
0/1
0/1
1/0
1/1
0/0

0/0
0/0
0/0
0/0
0/0
0/1
0/0
1/0
1/0
1/0

1/1
0/0
1/0
1/0
1/1
0/0
0/1
1/1
1/0
1/0

0/0
0/1
1/1
1/0
1/0
0/0
0/1
0/0
1/0
1/1

1/1
1/1
1/1
1/1
0/0
1/0
1/1
0/0
0/0
1/0

1/0
1/0
0/1
0/0
1/1
1/1
0/0
1/1
1/1
0/1

1/0
0/1
1/1
1/0
0/0
1/1
0/1
0/0
1/1
1/1

1/1
0/1
1/0
1/0
1/0
1/1
1/1
0/0
1/0
1/1

1/1
0/1
1/1
1/0
1/1
0/1
0/1
0/0
1/0
0/0

1/0
0/0
1/1
1/1
1/1
0/0
0/1
1/0
1/0
1/1

1/1
0/1
0/0
1/0
1/0
1/1
1/1
0/1
1/1
1/0

1/0
0/0
0/0
1/0
1/1
0/1
1/1
0/0
1/1
0/1

0/0
1/0
1/0
0/1
1/1
0/0
1/0
1/1
0/1
1/0

1/0
1/0
1/0
0/0
0/1
0/1
1/0
0/0
0/0
1/0

0/0
1/1
0/1
0/1
1/1
0/0
1/0
0/0
1/1
1/0

0/0
1/0
0/1
1/1
0/1
1/1
0/0
0/0
0/1
1/1

0/0
1/0
0/0
1/1
1/1
1/0
1/0
1/0
0/0
1/0

1/0
1/1
1/0
0/0
0/0
1/0
0/0
1/0
0/1
1/1
