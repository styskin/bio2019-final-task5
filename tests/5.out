14
21
18
7
50
29
33
11
38
10
28
4
46
45
30
16
2
17
19
23
8
9
32
1
34
43
31
37
26
20
3
6
12
48
47
35
22
27
40
5
49
41
15
25
42
13
44
24
36
39
